src/app/server/Mdns.cpp

```diff
@@ -247,22 +242,27 @@ CHIP_ERROR Advertise(bool commissionableNode)
 /// (Re-)starts the minmdns server
 void StartServer()
 {
-    ChipLogProgress(Discovery, "Start dns-sd server");
+    StartServer(false, false);
```

Comment:
Is this the right behavior for `StartServer` callers?  Should we perhaps not have this signature at all?  Seems easy to misuse....

Reply:
removed StartServer()